### ISSUE TITLE
[feature request] improved sort/group workspace variables

### ISSUE THREAD

**Issue by pdimens** [julia-vscode/julia-vscode#2839]
Is it possible to bring better organizational structure to the workspace view in the extension? Perhaps larger hierarchical categories such as `variables` that lists the variables you have assigned in your global scope, `packages` or `modules` that lists the items they bring into the namespace. Like the current collapsing view, this kind of additional structuring would allow a user to hide global variables if they aren't relevant, or only show global variables for a less cluttered view of the workspace.

**Comment by davidanthoff** [comment]
Yeah, something like that would be nice. Problem is probably whether we introduce a new level in the hierarchy at every node? That would also be cumbersome...

**Comment by pdimens** [comment]
In my mind's eye, I see 2-4 overarching groups like `global` `module`, etc. No idea what the internals of that would look like
